u2f-chrome-extension/jasmine/spec/SoftTokenHelperSpec.js

```diff
@@ -0,0 +1,365 @@
+      "should be able to transfer access to multiple accounts on different \
+       domains", function() {
+       });
+
+    // softokkenhelper to server communication
+    it(
+      "should sign in if the key handle matches a stored credential and a \
+       transferAccessMessage is stored with the same key handle", function() {
+         exampleTransferAccess.originalKeyHandle = keyHandle;
+         softTokenProfile.registrations = [exampleRegistration];
+         softTokenProfile.transferAccessChains = [exampleTransferAccess];
+         expect(softTokenProfile.transferAccessChains[0].originalKeyHandle)
+           .toEqual(keyHandle);
+         expect(softTokenProfile.registrations[0].keyHandle)
+           .toEqual(keyHandle);
+         var softTokenSignResponse = softTokenSignHandler.run(this.callback);
+         expect(softTokenSignResponse).toBe(true);
+         expect(this.callback)
+           .toHaveBeenCalledWith(expectedSoftTokenSignResponseMatchingKey);
+       });
+
+    it("should sign in if the key handle matches a stored credential and\
+        transferAccessMessages exist", function() {
+          softTokenProfile.registrations = [exampleRegistration];
+          softTokenProfile.transferAccessChains = [exampleTransferAccess];
+          var softTokenSignResponse = softTokenSignHandler.run(this.callback);
+          expect(softTokenSignResponse).toBe(true);
+          expect(this.callback)
+            .toHaveBeenCalledWith(expectedSoftTokenSignResponseMatchingKey);
+    });
+
+    it("should fail if no matching key or transferAccessMessage exists ",
+       function() {
+         softTokenProfile.transferAccessChains = [exampleTransferAccess];
+         var softTokenSignResponse = softTokenSignHandler.run(this.callback);
+         expect(softTokenSignResponse).toBe(true);
+         expect(this.callback)
+           .toHaveBeenCalledWith(expectedSoftTokenSignResponseNoMatchingKey);
+    });
+
+    it(
+      "should return a transferAccessResponse if a transferAccessMessage is \
+       stored instead of a normal key", function() {
+         softTokenProfile.transferAccessChains = [
+             exampleTransferAccess, exampleTransferAccessChain
+         ];
+         var softTokenSignResponse = softTokenSignHandlerForTransferAccess
+           .run(this.callback);
+         expect(softTokenSignResponse).toBe(true);
+         expect(this.callback)
+           .toHaveBeenCalledWith(expectedSoftTokenSignResponseTransferAccess);
+       });
+
+    it("should have normal credentials after delivering the transferAccessMessage\
+       to the server successfully", function() {
+         softTokenProfile.transferAccessChains = [exampleTransferAccess];
+         var softTokenSignResponse = softTokenSignHandlerForTransferAccess
+             .run(this.callback);
+         expect(softTokenProfile.transferAccessChains).toEqual([]);
+         expect(softTokenProfile.registrations).toEqual(expectedRegistration);
+    });
+
+    it(
```

Comment:
sometimes you start a string on the next line and sometimes on the first.  please be consistent